cibyl/sources/zuul/apis/rest.py

```diff
@@ -44,18 +44,22 @@ def __init__(self, session, host_url, verify):
         self._session = session
         self._session.verify = verify
 
-        if not host_url.endswith('/'):
-            host_url += '/'
+        if not host.endswith('/'):
+            host += '/'
```

Comment:
Why do we need to append '/'? urljoin does it by itself, no?

Reply:
If I remove this, the API URL is not correctly generated. For a host: localhost/zuul, I want: localhost/zuul/api, but get: localhost/api. Adding the slash at the end gives the correct result.